themes/default_theme/themeInit.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
```

Comment:
Every themes need this file ?
Maybe explains this in the readme (in /theme)
Any sample for VueJS or Angular ?